build.go

```diff
@@ -51,8 +52,23 @@ func main() {
 	}
 }
 
+func runCmd(args ...string) error {
+	var cmd *exec.Cmd
+	if len(args) > 1 {
+		cmd = exec.Command(args[0], args[1:]...)
+	} else {
+		cmd = exec.Command(args[0])
+	}
```

Comment:
I think args[1:] would return a nil slice, so it should be ok?

Reply:
I think the code would panic without the length check. Either way, if something is broken we can fix it.

Reply:
I think not:
https://play.golang.org/p/wxxS-Es9Qe6

Reply:
Do you have a specific suggestion as to how this code should change?

Reply:
```
	if len(args) > 1 {
		cmd = exec.Command(args[0], args[1:]...)
	} else {
		cmd = exec.Command(args[0])
	}
```
becomes
````
	cmd = exec.Command(args[0], args[1:]...)
```

Reply:
Right, that's what I had before and it was suggested I change it. I'll change it back.

Reply:
https://github.com/urfave/cli/pull/873/commits/c2d1a132082e3b02a219e61eeef49da364d2c315 ✔️

Reply:
Actually the suggestion was something else. There is no args is nil check.

Reply:
> Actually the suggestion was something else. There is no args is nil check.

Got it 👍  I'll add that

Reply:
Ah, I cant. It's not clear what you would return if there are no args.

Reply:
If we have the same signature as exec (arg, args...), it makes it impossible to have 0 arguments.

Reply:
Hey so, this is a PR for cleaning up a specific problem with the build system? I'm not etching code into a stone tablet here, we can change it if it's problematic. I have more important things to do than engage in _"what if this..."_ comments about the build system.

 I'm going to stop responding to this thread now, because it's starting to tank my mental health. If there's some more changes that should be made here, someone else can push to my branch and make them.

Reply:
So I've made the change, but I'd suggest not starting PRs if you don't have the will to finish them.

It wastes everyones time, your time on writing code that will not get merged as it's not finished, and our time for reviewing code that will not be driven to completion.